utils/flattenND.py

```diff
@@ -12,8 +12,8 @@ def __init__(self, array_width, *array_size):
             total_dims = total_dims * array_size[i]
             apparent_width = apparent_width * array_size[i]
 
-        self.in_ = self.input("in", width=array_width, size=array_size)
-        self.out_ = self.output("out", width=total_dims)
+        self.in_ = self.input("input_array", width=array_width, size=array_size, packed=True)
```

Comment:
if you use packed array, please use explicit_array as well for consistency with rest of source

Reply:
Done